tests/Adapters/Microsoft.Bot.Builder.Adapters.Twilio.Tests/TwilioHelperTests.cs

```diff
@@ -106,8 +106,8 @@ public void QueryStringToDictionary_Should_Return_Dictionary_With_Valid_Query()
             var builder = new StringBuilder(ValidationUrlString);
 
             var bodyString = File.ReadAllText(Directory.GetCurrentDirectory() + @"\files\NoMediaPayload.txt");
-            byte[] byteArray = Encoding.ASCII.GetBytes(bodyString);
-            MemoryStream stream = new MemoryStream(byteArray);
+            var byteArray = Encoding.ASCII.GetBytes(bodyString);
+            var stream = new MemoryStream(byteArray);
```

Comment:
Just a question, why change to var?

Reply:
Having `var` and a _explicit type_ (like _byte[]_) it's exactly the same in terms of performance, but it usage its encouraged for, for example, allowing to _adapt_ your code to changes in the methods you're calling. For example, if you left the _byte[]_ type but the _GetBytes_ method is refactored for returning another type, you must refactor your code; by the other hand, if you use `var`, this refactor wouldn't be necessary.

Reply:
BTW, the type is inferred in `compile time` (not in runtime).

Reply:
Super cool, I had no idea! Thanks!